### ISSUE TITLE
Information on the DAR Overview page

### ISSUE THREAD

**Issue by stevennewhouse** [Barts-Life-Science/Support#5]
Are the first 2 paragraphs needed now? These sections have no meaning and users will be presented with the questions they need to answer depending on their earlier responses.

​​If you are requesting access to Barts Health (BH) data in principle (before grant funding and other approvals are in place) then please complete Sections A, B/C, D, E and F in as much detail as possible and as applicable.

If this is an application for full approval, then all sections will need to be completed. Charges to access BH patient data will be made in accordance with NHS England’s guidelines and our own policies (https://bartslifesciences.org/precision-medicine/).

**Comment by stevennewhouse** [comment]
I also think this information can be presented when the relevant questions are asked. Presenting this information on the front page is overload?

 For all applications, please attach the result of the HRA algorithm (https://www.hra-decisiontools.org.uk/research/). Where relevant please attach your approved Clinical Audit (http://ceu:8080/) or Quality Improvement registration form (lifeqisystem.com) (please note these are internal links). We define the following project types:

 - Research: The activity is research if determined by the HRA Algorithm and will require appropriate ethical review.
- Clinical Audit (CA): Clinical audit measures the quality of care and services against agreed (national/local) standards, making improvements where necessary. Will require registration with CEU team.
- Quality Improvement (QI): Quality improvement supports making small changes that lead to measurable improvements for targeted services or patient populations. Will require registration with WeImprove team.
- Service Evaluation (SE): Service evaluations consider if existing or newly implemented services are effective. This process explores what is happening in a service as well as outcomes and experience for patients. Will require registration with CEU team.

**Comment by UddinRuz** [comment]
I had the same thoughts, the page is an overload of information.

**Comment by mhann90** [comment]
I agree with this. It's quite an overload of info for the first page.
